lodash.js

```diff
@@ -2494,14 +2494,19 @@
      *
      * @private
      * @param {Array} array The array to flatten.
-     * @param {boolean} [isDeep] Specify a deep flatten.
+     * @param {number} [maxDepth=1] The maximum recursion depth.
      * @param {boolean} [isStrict] Restrict flattening to arrays-like objects.
      * @param {Array} [result=[]] The initial result value.
      * @returns {Array} Returns the new flattened array.
      */
-    function baseFlatten(array, isDeep, isStrict, result) {
+    function baseFlatten(array, maxDepth, isStrict, result) {
+      maxDepth = maxDepth === undefined ? 1 : nativeMax(toInteger(maxDepth), 0);
       result || (result = []);
 
+      if (maxDepth < 1) {
```

Comment:
Naw just base functionality flatten should flatten at least 1 level. I knew a depth level [was coming for a while](https://github.com/mout/mout/commit/5957b66589f13f73de86b273f6d0040106cbd7a1) :P